.github/workflows/washboard.yml

```diff
@@ -49,6 +56,16 @@ jobs:
           name: washboard
           path: ./washboard-ui/washboard.tar.gz
 
+      - name: Upload Playwright Report
+        uses: actions/upload-artifact@v3
+        if: always()
```

Comment:
Nit: why use `if` if we always upload?

Reply:
Because a failed test (therefore a job) will product artifacts that are useful for troubleshooting such as screenshot, videos, and so on.

Reply:
Ah, `always()` forces this step to run, even if a previous step (the tests) fails. Gotcha